samples/csharp_dotnetcore/13.basic-bot/BasicBot.cs

```diff
@@ -210,7 +210,9 @@ private Activity CreateResponse(Activity activity, Attachment attachment)
         // Load attachment from file.
         private Attachment CreateAdaptiveCardAttachment()
         {
-            var adaptiveCard = File.ReadAllText(@".\Dialogs\Welcome\Resources\welcomeCard.json");
+            string[] paths = {".", "Dialogs", "Welcome", "Resources", "welcomeCard.json"};
```

Comment:
@sgellock  What is this?  A comment explaining why this is necessary would be good here. (this is weird)